### DISCUSSION TITLE
Karte im Verfolgermodus

### DISCUSSION THREAD

**Discussion by 1maxi2** [devemux86/cruiser#90] [Q&A] [answered]
Heute im Verfolgermodus gefahren.
Dabei stehen die Strassennamen am Kopf ? War das immer schon so oder mache ich einen Fehler ?
LG.

<img src="https://user-images.githubusercontent.com/124886589/226359708-023e6234-7eff-44b0-b283-bd5fdef222af.jpg" width="33%"/>

**Comment by devemux86** [comment, marked as answer]
You are using an online bitmap map.
Map rotation works best with vector maps, they can be downloaded / activated with the download manager in map menu:

---

Sie verwenden eine Online-Bitmap-Karte.
Die Kartendrehung funktioniert am besten mit Vektorkarten, die mit dem Download-Manager im Kartenmenü heruntergeladen / aktiviert werden können:

<img src="https://user-images.githubusercontent.com/3484020/226363992-a2f23789-6f35-4c16-b8f6-976f946e3f1e.jpg" width="50%"/>

**Reply by 1maxi2** [reply]
O.k. Danke. Werde ich ausprobieren.
